Composer/packages/client/src/components/CreationFlow/v2/CreationFlow.tsx

```diff
@@ -150,13 +154,21 @@ const CreationFlowV2: React.FC<CreationFlowProps> = () => {
     saveProjectAs(projectId, formData.name, formData.description, formData.location);
   };
 
+  const handleMigrate = (formData) => {
+    handleDismiss();
```

Comment:
handleDismiss sets the creation flow to CLOSE and then navigates home.  Is that all synchronous? This then sets the status to MIGRATE.  Do you want to only set the status to one state or does this need to go through the CLOSE state?